controllers/csm_controller.go

```diff
@@ -797,14 +797,18 @@ func (r *ContainerStorageModuleReconciler) SyncCSM(ctx context.Context, cr csmv1
 		if r.Config.IsOpenShift {
 			_ = drivers.RemoveVolume(&node.DaemonSetApplyConfig, drivers.ScaleioBinPath)
 		}
-
-		for _, env := range cr.Spec.Driver.Node.Envs {
-			if env.Name == "X_CSI_SDC_SFTP_REPO_ENABLED" {
-				if env.Value != "true" {
-					_ = drivers.RemoveInitVolume(&node.DaemonSetApplyConfig, drivers.SftpKeys)
+		if (cr.Spec.Driver.Node != nil) && cr.Spec.Driver.Node.Envs != nil {
+			for _, env := range cr.Spec.Driver.Node.Envs {
```

Comment:
One observation is that it seems we don't handle duplicate environment variables. For the X_CSI_SDC_SFTP_REPO_ENABLED environment variable, the first occurrence is considered, and for the other environment variables, the last occurrence is considered. Do we need to mention this behavior in the docs?